apps/cowswap-frontend/src/utils/iframeResizer.ts

```diff
@@ -0,0 +1,29 @@
+import { useEffect } from 'react';
+
+export function IframeResizer() {
```

Comment:
I find a bit strange that a component is in `utils` dir

Reply:
Has been moved to @cowprotocol/common-utils in https://github.com/cowprotocol/cowswap/pull/3255